### ISSUE TITLE
[a11y] 11.2.4.7 Focus visible

### ISSUE THREAD

**Issue by tschiebel** [owncloud/ios-app#1338]
audit: https://infinite.owncloud.com/s/qSvYucPltEYUhhc

https://github.com/BIK-BITV/BIK-App-Test/tree/main/Pr%C3%BCfschritte/de
https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf

Note: If the focus highlighting is done exclusively by a color change,
the contrast between the focused and unfocused state should be sufficient.
It is recommended that the contrast ratio is at least 3:1.

### Personal Space

- [x] For all buttons where the keyboard focus has been implemented by changing the background
background color, the keyboard focus is difficult to see. The
contrast between the focused and unfocused state is significantly below the recommended
the recommended contrast ratio of 3:1.
![image](https://github.com/owncloud/ios-app/assets/132444655/f60afe40-f394-4277-8608-d5b6599e82db)

> **Fix:** When focused, the secondary information row now uses the same color as the primary information row, providing better contrast:
> <img width="232" alt="Bildschirmfoto 2024-05-22 um 22 47 40" src="https://github.com/owncloud/ios-app/assets/639669/4b534dea-7b43-4327-b6e2-817fb3a2d1bd">
> The background color itself can't be changed as it is system-provided and used with the same contrast across the entire operating system.

### Menu, Shared with me, Shared with others

- [x] See commentary on Personal Space.

### Spaces

- [x] The space buttons have a barely visible keyboard focus.
![image](https://github.com/owncloud/ios-app/assets/132444655/eb51f907-3c28-4ee6-af1f-eacf564982f8)

**Comment by jesmrec** [comment]
Everything fixed here
